pypi_on_github_indexer/__main__.py

```diff
@@ -30,7 +30,10 @@ def parse_args():
                 ("target-dir", "Path in the index repository that is the PyPi root. We are "
                                "assuming GitHub Pages by default.", "docs"),
                 ("do-not-push", "Do not push to the index repo. Set this to whatever value "
-                                "to activate this option.", ""))
+                                "to activate this option.", ""),
+                ("repo-commit", "Hash of specific commit of repo to be installed", None),
```

Comment:
we don't want `repo-commits` as this is going to be part of an index. We want versioned packages.

Reply:
This is what I would also like, but in practice we may find that a version of the python package does not correspond to a release like in [`spark-connector-enterprise](https://github.com/src-d/gitbase-spark-connector-enterprise/blob/7fa60155dc271bb93fe4088ae6c3ab29d8ccbede/python/setup.py#L12), or we may need to point to a fork that fixes a bug before the PR is merged by the maintainer of a project like in [fastdtw](https://github.com/src-d/eee-commit-time-series/blob/156931ee422b4984c7d3f9011ee754e7059e12d7/requirements.txt#L1).

It is possible that I am not understanding how to properly make the package work. Could you please provide an example of how to install [this url from `gitbase-sce`](https://github.com/src-d/eee-commit-time-series/blob/156931ee422b4984c7d3f9011ee754e7059e12d7/requirements.txt#L16) so I can understand how to properly use the package?

Reply:
If it's one of our packages, it can be versioned. This should be treated the same way a PyPI package is normally treated and those have always proper versions.

In the case you're mentioning, you should add the `--extra-index-url  https://src-d.github.io/python-package-index/` line at the beginning of your `requirements.txt` file and then manage all packages by name instead of using the `git+https` url

This is mentioned in https://github.com/src-d/infrastructure/blob/master/docs/python-index.md#sourced-python-packages-index but it may not be clear enough. Let me know and I will further clarify it